### ISSUE TITLE
Wrong version of cards in decks

### ISSUE THREAD

**Issue by omfgitsmark** [mtgjson/mtgjson#560]
Quoted from discord:
> antz Today at 12:22 PM
> Hey all, first off thanks for making mtgjson! I've been working on an inventory / deck management web app (Angular / C#) for a little while now (hoping to get v1 out in a few weeks!).
> 
> Something very minor I've noticed, when I call https://www.mtgjson.com/json/decks/FlashOfFerocity_THB.json , one of the cards returned is 'Brazen Borrower', which I would expect. However the value for the property scryfallId is d42a57b2-fbe9-4024-a593-f6dc711a2e9d, which if you look it up, refers to the storybook alt art of the card (https://scryfall.com/card/eld/281/brazen-borrower-petty-theft).
> 
> As the Flash of Ferocity challenger deck doesn't contain the storybook alt art version of Brazen Borrower, I'm guessing this is wrong - any chance this can be changed or is this by design?
> 
> Thanks a lot! :thumbsup:
>

**Comment by ZeldaZach** [comment]
I have confirmed this to be an upstream issue re: @taw 

<img width="437" alt="Screenshot 2020-05-08 16 14 03" src="https://user-images.githubusercontent.com/7460172/81445500-f907d600-9146-11ea-8465-d5c200333a54.png">

**Comment by taw** [comment]
Oh right, so generally data we have is just card name / count / foiling of each card in the deck and sideboard.

The script can guess with nearly 100% accuracy which set the card is from based on a few simple rules.

Unfortunately if set has multiple versions of the same card, script just picks one at random. For pretty much whole history of Magic, this simply meant that we didn't have proper information which basics were in the deck, which is honestly not something anyone loses sleep over.

Now that there are multiple printings of each card in same set, I need to either add some logic or manually edit the lists to specify which one to use. 

[Here's an example of such annotations](https://github.com/taw/magic-preconstructed-decks/blob/838fc210fb4406d003a4f07bdb188322e2e073f8/data/theme/chk/Spiritbane.txt).

(those .txt files get transformed into .json with a lot of automatic processing)

Basically if you can tell me:

* which decks are affected
* which card (by number or some id) they should use

I'll change this and regenerate the jsons automatically.

**Comment by omfgitsmark** [comment]
well the only one that's been discovered so far is the one mentioned above.

**Deck:** Flash Of Ferocity
**Set:** Theros Beyond Death (THB)
**Current Card:** Brazen Borrower #281a 
**Correct Card:** Brazen Borrower #39

Would it be possible to have it grab the lowest numbered card (which is usually the normal version) or even the number without any extra characters (like the "a" in the above card number) unless the deck specifically is promoting that it has a certain special version of said card included?

**Comment by omfgitsmark** [comment]
Ok, It's a bit more than just the one wrong card. I have to go through and get the actual numbers when I have some time, but here's the list given to me:

- Final Adventure deck: Foulmire Knight // Profane Insight, Murderous Rider // Swift End, Blacklance Paragon, Order of Midnight // Alter Fate
 - Flash of Ferocity - Thassa's Intervetion, Wavebreak Hippocamp, Brazen Borrower // Petty Theft
 - Allied Fires - Fae of Wishes // Granted, Thirst for Meaning
 - Elspeth (Planeswalker Deck) - Daxos Blessed By the Sun, Taranika, Akroan Veteran
 - Askiok (Planeswalker Deck)  -  Gravebreaker Lamia
 - Rowan, Fearless Sparkmage: Ardenvale Tactician // Dizzying Swoop
 - Faerie Schemes - Animating Faerie // Bring to Life
 - Knights' Charge - Foulmire Knight // Profane Insight, Order of Midnight // Alter Fate

**Comment by taw** [comment]
[I created issue in upstream repository](https://github.com/taw/magic-search-engine/issues/141). Currently 199 non-basic cards match multiple printings in same set, so this needs more systematic approach.

If you want to help, use that list, and check manually which card should be used (give me numbers for each conflict).
You can ignore cards with ★ or †, I'll have indexer deal with them automatically.

**Comment by taw** [comment]
@omfgitsmark With new heuristics all those issues are fixed, but it would be nice if someone went through https://mtg.wtf/deck and checked if everything is correct.

(like I've noticed some full art basics in some decks, that's definitely wrong; I didn't notice any other issues)

**Comment by taw** [comment]
[As far as I can tell, the only remaining issues are in DKM and CM2 and they're trivial](https://github.com/taw/magic-search-engine/issues/141#issuecomment-629601118) so this probably counts as fixed.
